### ISSUE TITLE
Error purging outdated node reference

### ISSUE THREAD

**Issue by holtzp** [pwm-project/pwm#518]
If PWM temporarily loses connectivity to the LDAP server, it perpetually reports the below error until PWM is restarted, months after LDAP connectivity has been restored:

![PWM Error](https://user-images.githubusercontent.com/43244100/70641148-c347db00-1c02-11ea-9128-14a4a000958d.PNG)

I don't know if this is just cosmetic, or if PWM never attempts to reconnect to the LDAP server to purge the outdated node reference until restart.

Thanks!
Paul

**Comment by mayyit** [comment]
On v2.0.3 and have similar reports. Both censored databases references the image are the same:

![image](https://user-images.githubusercontent.com/26198665/188878024-1e2ad219-5a37-4a0e-8e1e-9a73e641e68a.png)

I'm going to try to disable the node service and see how PWM behaves:

![image](https://user-images.githubusercontent.com/26198665/188878756-c8bec233-0966-47b7-be84-1b879d7f0a8c.png)

**Comment by jrivard** [comment]
Yep, looks like a bug, the node service should clear out that caution once it reconnects.
